services/softlayer_network_storage.go

```diff
@@ -38,20 +38,33 @@ func (slns *softLayer_Network_Storage_Service) CreateIscsiVolume(size int, locat
 		return datatypes.SoftLayer_Network_Storage{}, errors.New("Cannot create negative sized volumes")
 	}
 
-	iscsiVolumeItemId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
-	if err != nil {
+	sizeItemPriceId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
+	iopsItemPriceId, err := slns.getPerformanceStorageItemPriceIdByIops(size)
+
+	/*if err != nil {
 		return datatypes.SoftLayer_Network_Storage{}, err
-	}
+	}*/
 
 	order := datatypes.SoftLayer_Product_Order{
 		Location:    location,
-		ComplexType: "SoftLayer_Container_Product_Order",
+		ComplexType: "SoftLayer_Container_Product_Order_Network_PerformanceStorage_Iscsi",
+		OsFormatType: datatypes.OsFormatType{
+			Id:      12,
+			KeyName: "LINUX",
+		},
 		Prices: []datatypes.SoftLayer_Item_Price{
 			datatypes.SoftLayer_Item_Price{
-				Id: iscsiVolumeItemId,
+				Id: sizeItemPriceId,
+			},
+			datatypes.SoftLayer_Item_Price{
+				Id: iopsItemPriceId,
+			},
+			datatypes.SoftLayer_Item_Price{
+				Id: BLOCK_ITEM_PRICE_ID,
 			},
 		},
-		PackageId: NETWORK_STORAGE_PACKAGE_ID,
+		PackageId: NETWORK_PERFORMANCE_STORAGE_PACKAGE_ID,
+		Quantity:  1,
 	}
 
 	productOrderService, _ := slns.client.GetSoftLayer_Product_Order_Service()
```

Comment:
Why are we ignoring `err` here? We should never do so. If there is an `err` then we should fail.